suites/krbd/rbd/tasks/rbd_fio.yaml

```diff
@@ -3,7 +3,7 @@ tasks:
 - ceph: null
 - rbd_fio:
     client.0:
-       fio-version: 2.2.9
+       fio-version: 2.2.11
```

Comment:
@dillaman sure I can update the default there, but we can keep this versions unchanged in yaml, it will be helpful when testing new/old fio version using overrides without any other modification.